lib/grizzly/commands/command.ex

```diff
@@ -285,11 +285,42 @@ defmodule Grizzly.Commands.Command do
         {key, value}, m -> Keyword.put(m, key, value)
         {key, value, value2}, m -> Keyword.put(m, key, {value, value2})
       end)
+      |> Enum.map(fn tuple -> transform(tuple) end)
+      |> List.flatten()
     else
       meta
     end
   end
 
+  def transform({_key, nil} = tuple), do: tuple
+  def transform({:transmit_channel, _value} = tuple), do: tuple
+  def transform({:transmission_time, value}), do: {:transmit_time, value}
+  def transform({:route_changed, :changed}), do: {:route_changed, true}
+  def transform({:route_changed, :not_changed}), do: {:route_changed, false}
+
+  def transform({:rssi, values}) do
+    actual_rssis = Enum.filter(values, fn rssi -> rssi != :not_available end)
+    average = calculate_average(actual_rssis)
+    [{:rssi_hops, actual_rssis}, {:rssi_dbm, average}, {:rssi_4bars, calculate_bars(average)}]
+  end
+
+  def transform({:last_working_route, [route, _r2, _r3, _r4, transmit_speed]}) do
+    [{:last_working_route, route}, {:transmit_speed, transmit_speed}]
+  end
+
+  defp calculate_bars(number) when number >= -50, do: 4
+  defp calculate_bars(number) when number >= -60, do: 3
+  defp calculate_bars(number) when number >= -70, do: 2
+  defp calculate_bars(number) when number >= -80, do: 1
+  defp calculate_bars(_number), do: 0
+
+  defp calculate_average(numbers) when numbers == [], do: 0
+
+  defp calculate_average(numbers) do
+    (Enum.sum(numbers) / length(numbers))
+    |> Float.round(2)
```

Comment:
```suggestion
    Enum.sum(numbers) / length(numbers)
```

I'm not sure the right answer on this. If the user is expecting an number then I'd let them choose the precision rather than rounding here. On the other hand, I think we pass around signal strength values pretty much everywhere else as integers, so calling `round/1` on the result to make this an integer too seems good as well.